django-test-suite: "Permission errors are not swallowed" for file_uploads.tests.DirectoryCreationTests.test_readonly_root

Just noticed as the consistent sole failure after moving to Google Cloud Build:
[Full build listing](https://source.cloud.google.com/results/invocations/97db129d-7b50-4850-b255-b443b595ef27/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-spanner-django%2Fpresubmit%2Fpresubmit/log)

and in particular the [test failure](https://source.cloud.google.com/results/invocations/7f7bfc9c-baf8-4030-a870-217e52f331c7/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-spanner-django%2Fpresubmit%2Fworker_1/log)
```shell
======================================================================
FAIL: test_readonly_root (file_uploads.tests.DirectoryCreationTests)
Permission errors are not swallowed
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmpfs/src/github/python-spanner-django/django_tests/django/tests/file_uploads/tests.py", line 548, in test_readonly_root
    self.obj.testfile.save('foo.txt', SimpleUploadedFile('foo.txt', b'x'), save=False)
AssertionError: PermissionError not raised

----------------------------------------------------------------------
Ran 50 tests in 133.498s

FAILED (failures=1, skipped=15)
```

--- a/django_spanner/features.py
+++ b/django_spanner/features.py
@@ -278,9 +278,6 @@ class DatabaseFeatures(BaseDatabaseFeatures):
         # This test isn't isolated on databases like Spanner that don't
         # support transactions: https://code.djangoproject.com/ticket/31413
         'migrations.test_loader.LoaderTests.test_loading_squashed',
-        # Migrations data  persistance issue to be investigated:
-        # https://github.com/googleapis/python-spanner-django/issues/408
-        'migration_test_data_persistence.tests.MigrationDataNormalPersistenceTestCase.test_persistence',
     )
     # Kokoro-specific skips.
     if os.environ.get('KOKORO_JOB_NAME'):

--- a/django_spanner/features.py
+++ b/django_spanner/features.py
@@ -278,13 +278,16 @@ class DatabaseFeatures(BaseDatabaseFeatures):
         # This test isn't isolated on databases like Spanner that don't
         # support transactions: https://code.djangoproject.com/ticket/31413
         'migrations.test_loader.LoaderTests.test_loading_squashed',
-        # "Permission errors are not swallowed":
-        # https://github.com/googleapis/python-spanner-django/issues/407
-        'file_uploads.tests.DirectoryCreationTests.test_readonly_root',
         # Migrations data  persistance issue to be investigated:
         # https://github.com/googleapis/python-spanner-django/issues/408
         'migration_test_data_persistence.tests.MigrationDataNormalPersistenceTestCase.test_persistence',
     )
+    # Kokoro-specific skips.
+    if os.environ.get('KOKORO_JOB_NAME'):
+        skip_tests += (
+            # os.chmod() doesn't work on Kokoro?
+            'file_uploads.tests.DirectoryCreationTests.test_readonly_root',
+        )
 
     if os.environ.get('SPANNER_EMULATOR_HOST', None):
         # Some code isn't yet supported by the Spanner emulator.

